docs/lessons/09-interitance.md

```diff
@@ -0,0 +1,107 @@
+# 09 - Inheritance & overriding
+<Teacher name="Alina"></Teacher>
+
+## Goals
+- Understand what is Inheritance and why do we need it 
+- Understand what is Overriding
+- Understand what is difference between Overriding and Overloading
```

Comment:
I think `the` is missing between `is` and `difference` here.

Reply:
Hate articles ...